commit e002be7d335cf3c299e2260f3d1ed6ac73f7dbd3
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date:   Mon Nov 11 15:10:09 2013 -0500

    avoid warning: variable 'rv' set but not used [-Wunused-but-set-variable]
    
    This patch allows the build to proceed cleanly when using
    -Wunused-but-set-variable

diff --git a/src/nsEnigMsgCompose.cpp b/src/nsEnigMsgCompose.cpp
index 2bc3f7a..bd4928a 100644
--- a/src/nsEnigMsgCompose.cpp
+++ b/src/nsEnigMsgCompose.cpp
@@ -77,10 +77,9 @@ nsEnigMsgCompose::nsEnigMsgCompose()
 
 nsEnigMsgCompose::~nsEnigMsgCompose()
 {
-  nsresult rv;
 #ifdef FORCE_PR_LOG
   nsCOMPtr<nsIThread> myThread;
-  rv = ENIG_GET_THREAD(myThread);
+  ENIG_GET_THREAD(myThread);
   DEBUG_LOG(("nsEnigMsgCompose:: >>>>>>>>> DTOR(%p): myThread=%p\n",
          this, myThread.get()));
 #endif
